### ISSUE TITLE
External plugin making topic subscription incorrectly

### ISSUE THREAD

**Issue by AaronLionsheep** [FlyingDiver/Indigo-MQTT#10]
I believe I have come across a bug in the way subscriptions are added when an external plugin makes a call like:
```
props = {
    'topic': subscription,
    'qos': 0
}
mqtt.executeAction("add_subscription", deviceId=self.getBrokerId(), props=props)
```

This results in the broker having the following subscriptions:
![Screen Shot 2020-03-01 at 11 57 16 AM](https://user-images.githubusercontent.com/3443748/75629888-156e1c00-5bb4-11ea-91f1-38ab957358ed.png)

The `0:test` was added manually using the entry form above. Attempting to save the broker config results in an error:

```
   MQTT Connector Error            Error in plugin execution UiValidate:

Traceback (most recent call last):
  File "plugin.py", line 498, in validateDeviceConfigUi
ValueError: invalid literal for int() with base 10: 'i'
```

Which I believe is cause by the code expecting the entries in this list to be of the form `qos:topic` 

The config appears to add topics by eventually calling https://github.com/FlyingDiver/Indigo-MQTT/blob/184fc80af8a416638b4bf9a7a8bba869b1f4d4e4/MQTT%20Connector.indigoPlugin/Contents/Server%20Plugin/plugin.py#L358

and the `add_subscription` action from a plugin is handled by another function that adds a subscription without the proper qos prefix https://github.com/FlyingDiver/Indigo-MQTT/blob/184fc80af8a416638b4bf9a7a8bba869b1f4d4e4/MQTT%20Connector.indigoPlugin/Contents/Server%20Plugin/plugin.py#L689

You have to manually remove these subscriptions before the broker device will work again and allow you to save the config. The external plugin can make the subscriptions again (and they work), but it will break as soon as the broker config is opened again.

**Comment by FlyingDiver** [comment]
Probably can’t fix until I get home in three weeks. Well, I could do a code update but have no way to test it remotely.

**Comment by FlyingDiver** [comment]
Fixed for 0.3.3.
